xcp/cpiofile.py

```diff
@@ -1140,10 +1140,14 @@ def xzopen(cls, name, mode="r", fileobj=None, compresslevel=6):
 
         if fileobj is not None:
             raise CompressionError("passing fileobj not implemented for LZMA")
-        else:
-            # FIXME: not compatible with python3 API
-            fileobj = lzma.LZMAFile(name, mode, options={'level': compresslevel, 'dict_size': 20 })
-
+        kwargs = {}
+        if sys.version_info < (3, 0):
+            # Old code passed `dict_size: 20` but pyliblzma does not support such sizes:
```

Comment:
And this too... Describing old bugs in comments just makes the code harder to read. That is what the commit history is for.

Reply:
Thanks, agreed, done!